glitch/apps.py

```diff
@@ -154,3 +171,40 @@ def glitch_video(input_path: str, output_path: str) -> None:
     reader.wait()
     writer.stdin.close()
     writer.wait()
+
+
+def swap_blocks_static(width: int, height: int, options: dict) -> tuple:
```

Comment:
python type hinting is a bit weird, so the builtint containers types are not used directly. `int`, `float`, `bool`, `str` and `bytes` are used normally, but for containers we use:
```python2
from typing import Dict, Tuple, List
def foo(xs: List[int], options: Dict[str, float]) -> Tuple[int,int,str]:
  ...
```

Reply:
also, its actually returning a Dict